errors when using paths:

`NameError: name 'glob' is not defined`

`import glob`, will not solve this issue, since glob refers to path of some kind.

The following syntax has issues in python 3.6:

this was missing from the import statements:

`from glob import glob`



